Trading/Exchange/bybit/bybit_exchange.py

```diff
@@ -155,7 +155,7 @@ async def create_order(self, order_type: trading_enums.TraderOrderType, symbol:
                                           side=side, current_price=current_price,
                                           params=params)
 
-    async def _create_market_stop_loss_order(self, symbol, quantity, price, side, current_price, params=None) -> dict:
+    async def _create_market_stop_loss_order(self, symbol, quantity, price, side, current_price, tag: str = None, params=None) -> dict:
```

Comment:
Why adding tag support?

Reply:
According to our checks, tag is not a unified ccxt argument, therefore if you want to pass it, you can do it with `order.exchange_creation_params` when necessary but it should not be in the unfirm api

Reply:
I removed it, with the coming order storage it wont be necessary for me anymore to store the tag on the exchange.